_posts/2023-06-05-pride-spotlight-sophie-wilson.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: post
+title: "US-RSE Pride Month Spotlight - Sophie Wilson"
+tags: [dei, pride]
+author: Lezlie Espana
```

Comment:
I wrote this one, FYI

Reply:
Sorry! I got things mixed up. Fixed now.